Source/Tests/FEXLoader.cpp

```diff
@@ -299,7 +333,7 @@ int main(int argc, char **argv, char **const envp) {
   if (!ExecutedWithFD && !FEXFD && !FHU::Filesystem::Exists(Program.ProgramPath)) {
     // Early exit if the program passed in doesn't exist
     // Will prevent a crash later
-    fmt::print(stderr, "{}: command not found\n", Program.ProgramPath);
+    fextl::fmt::print("{}: command not found\n", Program.ProgramPath);
```

Comment:
Why is `stderr` dropped here and in the other uses in this file? Don't we just need another `print` overload to support the original semantics?

Reply:
Resolved by adding FILE handlers just like fmt.
Will need to check later if fwrite itself does memory allocations but for the sake of pushing this through I'll let it be for now.